src/jest-recorder.js

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const path = require('path');
+const stream = require('stream');
+const child_process = require('child_process');
+
+const jestRecorder = () => {
+    const fd = fs.openSync(path.join(__dirname, '..', 'console.log'), 'w+', 0o644);
+    const ws = stream.Writable({
+        write (data, encoding, cb) { fs.write(fd, data, undefined, encoding, cb) },
+    });
+    const child = child_process.spawn('jest', ['--no-colors', ...process.argv.slice(2)]);
```

Comment:
@FriggaHel why no colors? is this affecting the output on the console.log?

Reply:
@vrunoa this removes the control characters generating colors in the terminal. It remove needs to clean the file or evaluate ANSI control character.